files/private-chef-cookbooks/private-chef/templates/default/bookshelf.erb

```diff
@@ -1,185 +0,0 @@
-#!/bin/bash
```

Comment:
Given that this is replaced by the generated version, have you been able to verify that chef-server-ctl stop/start works correctly with the replacement? (It should, but just in case...)